### PR TITLE
Use assert methods from unittest instead of raw assert in tests

### LINKED ISSUE #2948: Use assert methods from unittest instead of clean "assert" in tests
SSIA

### PR DESCRIPTION
**Pull request by sebwoj**
Fixes #2948

Signed-off-by: Sebastian Wojciechowski <s.wojciechowski89@gmail.com>

### CODE REVIEW

**Comment by bowlofeggs** [comment]
The ```f30-pydocstyle``` test actually passed - the CI system just failed to report it to GitHub.

File: bodhi/tests/server/services/test_updates.py [hunk 1]
```diff
@@ -3624,12 +3623,8 @@ def test_stable_request_when_stable(self, publish, *args):
 
         self.assertEqual(resp.json['update']['status'], 'stable')
         self.assertEqual(resp.json['update']['request'], None)
-        try:
-            publish.assert_called_with(
-                topic='update.request.stable', msg=mock.ANY)
-            assert False, "request.stable fedmsg shouldn't have fired"
-        except AssertionError:
-            pass
```
Comment (bowlofeggs) [inline bodhi/tests/server/services/test_updates.py:3632]: Hahahahaha, whaaaaaat was going on there?

**Review by bowlofeggs — APPROVED** [review]
Thanks!

File: bodhi/tests/server/services/test_updates.py (hunk 1 above)
Comment (sebwoj) [reply bodhi/tests/server/services/test_updates.py:3632]: Yeah, there are some places like this one in Bodhi. Take a look at `validate_updateid` :D

**Pull request merged**